mlflow/langchain/__init__.py

```diff
@@ -1056,6 +1060,12 @@ def autolog(
             We do not guarantee classes specified in this list can be logged as a model, but tracing
             will be supported. Note that all classes within the list must be subclasses of Runnable,
             and we only patch `invoke`, `batch`, and `stream` methods for tracing.
+        allow_async: If ``True``, enable autologging on langchain model async methods. Default to
+            ``False``. Note that if this is enabled and any of the following options are turned on
+            (log_models, log_input_examples, log_model_signatures, or log_inputs_outputs),
+            autologging may cause blocking in the model inference process because autologging itself
```

Comment:
```suggestion
            autologging may slow down the model inference process because autologging itself
```

can we replace `blocking` with slow down?